R/reporter-junit.R

```diff
@@ -1,8 +1,12 @@
 #' @include reporter.R
 NULL
 
+#' Strip illegal chars from a context or test_that description string.
```

Comment:
Can you please use regular comments here? I think you should also include a small test so that it's not accidentally broken in the future.

Reply:
I dropped the roxygen quotes from the comments, and wasn't sure how you would like a test case, but I made an attempt that parses the xml output and checks the "name" attributes in the "testsuite" elements.